cnf-tests/testsuites/e2esuite/test_suite_test.go

```diff
@@ -4,75 +4,79 @@
+	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/security"                   // this is needed otherwise the security test won't be executed
+	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/sro"                        // this is needed otherwise the sro test won't be executed
+	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/vrf"                        // this is needed otherwise the vrf test won't be executed
 	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/xt_u32"                     // this is needed otherwise the xt_u32 test won't be executed
 	_ "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/1_performance" // this is needed otherwise the performance test won't be executed
 	_ "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/4_latency"     // this is needed otherwise the performance test won't be executed
 
 	_ "github.com/k8snetworkplumbingwg/sriov-network-operator/test/conformance/tests"
-	sriovNamespaces "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/namespaces"
 	_ "github.com/metallb/metallb-operator/test/e2e/functional/tests"
 	_ "github.com/openshift/ptp-operator/test/conformance/ptp"
 
-	perfUtils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
-
-	sriovClean "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/clean"
-	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/clean"
-	testclient "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
-	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/discovery"
-	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/features"
 	testutils "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/utils"
-	perfClean "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/clean"
-	ptpClean "github.com/openshift/ptp-operator/test/utils/clean"
 
-	numaserialconf "github.com/openshift-kni/numaresources-operator/test/e2e/serial/config"
 	_ "github.com/openshift-kni/numaresources-operator/test/e2e/serial/tests"
 
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
 )
 
 // TODO: we should refactor tests to use client from controller-runtime package
 // see - https://github.com/openshift/cluster-api-actuator-pkg/blob/master/pkg/e2e/framework/framework.go
 
 var (
-	junitPath          *string
-	reportPath         *string
-	skipTestNSCreation bool
+	junitPath  *string
+	reportPath *string
 )
 
+var suiteFixtureMap = map[string]features.SuiteFixture{
```

Comment:
I think we are missing s2i no?

Reply:
we only have here the features that we set/delete something for them before running the tests. is there a setup for s2i?

Reply:
that's what we currently have, i don't see anything for s2i https://github.com/openshift-kni/cnf-features-deploy/blob/57a05afbe836e9210ad638f6abe65f72b569148a/cnf-tests/testsuites/e2esuite/test_suite_test.go#L103